libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/routefetch/RoutesAction.kt

```diff
@@ -0,0 +1,17 @@
+package com.mapbox.navigation.dropin.component.routefetch
+
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.geojson.Point
+import com.mapbox.navigation.base.route.NavigationRoute
+import com.mapbox.navigation.dropin.model.Destination
+
+sealed class RoutesAction {
+    data class SetDestination(val destination: Destination?) : RoutesAction()
+    object FetchAndSetRoute : RoutesAction()
+    object StartNavigation : RoutesAction()
+    object StopNavigation : RoutesAction()
```

Comment:
These should be part of `NavigationStateViewModel` actions?

Reply:
They depend on fetchRoute.

Reply:
^^ https://github.com/mapbox/mapbox-navigation-android/pull/5506/files/cb4796ff500924d993d3925d1f403d03547b38e2#r822093458